docs/docs/adding-page-transitions-with-plugin-transition-link.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Using transition-link for Page Transitions
+---
+
+This guide will cover how to use `gatsby-plugin-transition-link` to animate transitions between pages on your Gatsby site.
+
+## Overview
+
+The `TransitionLink` component is a simple way of describing a page transition via props on a Link component. It works with many animation libraries, like [react-pose](https://popmotion.io/pose/), [gsap](https://greensock.com/), [animejs](https://animejs.com/), and many others.
+
+Note that currently, as the plugin is based on link navigation, transitions when navigating with the browser buttons are not supported.
+
+For other page transition options, see the [overview on adding page animations](/docs/adding-page-transitions).
+
+## Getting started
+
+First, install the plugin:
+
+```shell
+yarn add gatsby-plugin-transition-link
+```
+
+```shell
+npm install --save gatsby-plugin-transition-link
```

Comment:
We prefer `npm` over `yarn` for docs, so you can list just the second install instruction here.

Reply:
Gotcha!